### ISSUE TITLE
Allow cell padding to be changed

### ISSUE THREAD

**Issue by tdwright** [tdwright/contabs#34]
The default cell padding is one character left and right. This can look cramped for some data sets.

Can we add a table-level property to allow the padding on the left and right side of the heading/data to be set?

I would imagine we would keep the padding symmetrical and apply one value to all columns.

I think at this stage we would ignore vertical spacing between rows, although this could be addressed in a separate issue.

**Comment by Ravikc** [comment]
This sounds good. Can I start working on it ?

**Comment by tdwright** [comment]
@Ravikc Thanks for your interest. It's all yours! 😄 

If you have any thoughts on how you plan to implement this, I'd be more than happy to discuss them here.

**Comment by Ravikc** [comment]
I am thinking about using custom padding only when `ContentAlignment `and `HeaderAlignment `are both set to `Center`. Later we can figure out how the padding behaves when the either one is not set to `Center` ?

**Comment by Ravikc** [comment]
How do you want me to handle it when applying the user defined `Padding` value exceeds the `MaxWidth` property of a `Column` class ?

**Comment by tdwright** [comment]
The `MaxWidth` property of a column is read-only. It is [currently calculated](https://github.com/tdwright/contabs/blob/master/ConTabs/Column.cs#L21-L35) from the maximum width of the actual content - ignoring the current implicit padding of 1 character.

As this property is primarily used whilst rendering (i.e. it's not of much direct value to an end-user), I suppose we could alter its behaviour without too much of a headache.

On the other hand, it feels to me that incorporating the padding into the `MaxWidth` property is going to tangle things too much. Best to keep the concerns separated. For this reason, I suggest that `Padding` is applied in `OutputBuilder.cs`. In other words, we'd change lines like this:

```C#
sb.Append(" " + table.HeaderAlignment.ProcessString(col.ColumnName, col.MaxWidth) + " " + style.Wall);
```
To something like this:
```C#
sb.Append(Padding + table.HeaderAlignment.ProcessString(col.ColumnName, col.MaxWidth) + Padding + style.Wall);
```
Where `Padding` is defined in `Table` as something like:
```C#
new String(' ',PadWidth);
```
So what we end up with in terms of the total width of the column is Padding + MaxWidth + Padding. For example:
```
+-----------------+
|     Name        |
+-----------------+
|     Mercury     |
|     Venus       |
|     Earth       |
+-----------------+

W<-P-><--M--><-P->W
   5     7     5
```
Where P = padding, M = max width, and W = wall.

What do you think?

**Comment by tdwright** [comment]
Closed by #38 - thanks @Ravikc!
